### ISSUE TITLE
Create some sort of animated gif for the readme 

### ISSUE THREAD

**Issue by freemo** [opennars/opennars#141]
A nice visually pleasing animated gif on the readme goes a long way. Even just as eye candy it can help give a nice professional look to the readme. But it can also be a nice way to show off the app to a passer-by. See this project for an example: https://github.com/Syncleus/aparapi

**Comment by TonyLo1** [comment]
Here is a link to a gif: https://1drv.ms/u/s!AsfcIQ1U-xtujvBhOzyhjR_ifc2B7g
I think we need an images directory in the repo to store these. Not sure of best place to locate it.

**Comment by freemo** [comment]
Con't it play pong? maybe make a gif out of that?

**Comment by TonyLo1** [comment]
Good idea - we could have a number of examples: pong, microworld, and test chamber. That would be nice eye-candy

**Comment by freemo** [comment]
Maybe create a GIF that shows all of the eye candy sort of examples at once
(split screen 2x2 grid maybe?)

**Comment by TonyLo1** [comment]
yes - that would work

**Comment by TonyLo1** [comment]
@freemo , Are you aware of any standards regarding size of gifs for placement on github

**Comment by freemo** [comment]
As long as it fits the space on github your fine. The width is fixed but I
dont know what it is off hand. probably 800px max if i had to guess
